desktop/main.js

```diff
@@ -179,7 +179,6 @@ const mainWindow = (() => {
             }
 
             if (ELECTRON_ENVIRONMENT.isDev()) {
-                require('dotenv').config();
```

Comment:
⚠️ I may have removed this by mistake
Will check it out tomorrow. 

Local development works for me (no .env file), but probably doesn't use the web proxy 
My idea here was that `dotenv` is used during bundling (webpack), but I don't think it's getting applied here
(local dev doesn't use webpack but calls `main.js` directly)